src/content/ui-api/observability/prometheus/volume-size/index.md

```diff
@@ -0,0 +1,83 @@
+---
+linkTitle: Prometheus Volume Size
+title: Prometheus Volume Sizing
+description: A guide explaining how to resize the Prometheus persistent volume to fit your needs
+weight: 50
+menu:
+  main:
+    identifier: uiapi-observability-prometheusvolumesize
+    parent: uiapi-observability-prometheus
+user_questions:
+  - How can I downsize the persistent volume of the Giant Swarm managed Prometheus?
```

Comment:
Consider (author's choice):
```suggestion
  - How can I shrink the persistent volume of the Giant Swarm managed Prometheus?
```
or `reduce the size of`.

Just because that might be how I'd search for the topic rather than downsize.